src/routes/log.ts

```diff
@@ -365,12 +365,18 @@ router.route('/startRPCCapture').get(async function (req: Request, res: Response
 })
 router.route('/stopRPCCapture').get(async function (req: Request, res: Response) {
   if (!CONFIG.statLog) {
-    return res.json({ message: 'Interface stats recording already stopped' }).status(304)
+    return res.json({ message: 'Interface stats recording already stopped' }).status(204)
   }
   debug_info.interfaceRecordingEndTime = Date.now()
   CONFIG.statLog = false
   res.json({ message: 'RPC interface recording disabled' }).status(200)
 })
+router.route('/countRPCCapture').get(async function (req: Request, res: Response) {
+  if (!CONFIG.statLog) {
+    return res.json({ message: 'Interface stats recording disabled' }).status(204)
+  }
+  res.json(getInterfaceStatCounts()).status(200)
+})
```

Comment:
I don't see this method mentioned in the readme. Look's like it was added in this PR and is calling a function that doesn't exist.